en/docs/apis/api-concepts.md

```diff
@@ -1,30 +1,30 @@
 # API Concepts
-Choreo API management capabilities allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. Here are some basic concepts which will help you understand API management in Choreo as you create and manage your APIs in the [Choreo Console](https://console.choreo.dev/login/).
+Choreo API management capabilities allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. This topic presents some basic concepts to help you understand API management in Choreo as you create and manage your APIs in the [Choreo Console](https://console.choreo.dev/login/).
  
 ## API
 An API is an intermediate layer that acts as a communication protocol between a consumer and a service, simplifying the consumption of the service. In addition to hiding the underlying implementation details of a service, an API provides a secure, controlled, and well-documented approach to access an exposed service.
 
 ## API format
-OpenAPI Specification (OAS, a.k.a Swagger) format is the underlying representation of an API in Choreo. You can import an API using a Swagger definition in the Choreo Console. Additionally, a user can view, edit, import, or download an API definition in OAS format via the Choreo Console.
+The OpenAPI Specification (OAS, also known as Swagger) format is the underlying representation of an API in Choreo. You can import an API using a Swagger definition in the Choreo Console. Additionally, a user can view, edit, import, or download an API definition in OAS format via the Choreo Console.
 
 ## API resource path and HTTP methods
 An API comprises one or more resources, each of which has a URI. An API resource has a set of HTTP methods that operates on it. The supported HTTP methods are `GET`, `POST`, `PUT`, `DELETE`, `PATCH`, `HEAD`, and `OPTIONS`.
 
 ## API lifecycle
-An API lifecycle comprises the stages an API goes through from creation to retirement. API lifecycle is independent of its backend service. The lifecycle states are: `Created`, `Prototyped`, `Published`, `Deprecated`, `Blocked`, `Deployed`, and `Retired`.
+An API lifecycle comprises the stages an API goes through from creation to retirement. An API lifecycle is independent of its backend service. The lifecycle states are: `Created`, `Prototyped`, `Published`, `Deprecated`, `Blocked`, `Deployed`, and `Retired`.
```

Comment:
Should this be "The"

Reply:
Since we can use either "An" or "The" here, left it as is.

Reply:
An API has only one API lifecycle right?